relay-pii/src/convert.rs

```diff
@@ -1608,6 +1610,33 @@ THd+9FBxiHLGXNKhG/FRSyREXEt+NyYIf/0cyByc9tNksat794ddUqnLOg0vwSkv
         assert_annotated_snapshot!(data);
     }
 
+    #[test]
+    fn test_span_cookie() {
+        let mut data = Event::from_value(
+            serde_json::json!({
+                "contexts": {
+                    "trace": {
+                      "data": {
+                        "http.request.header.cookie": "session=hello-secret",
+                      },
+                      "type": "trace"
+                    }
+                },
+                "spans": [{
+                      "data": {
+                        "http.request.header.cookie": "session=hello-secret",
+                      },
+                    }]
```

Comment:
```suggestion
                }]
```

Formatting seems off here.